packages/flutter/test/material/tabs_test.dart

```diff
@@ -2892,6 +2892,78 @@ void main() {
     // No other tab got instantiated during the animation.
     expect(log, <String>['init: 0', 'init: 3', 'dispose: 0']);
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/10969.
```

Comment:
This test appears to pass on master even without your changes applied.

Reply:
https://github.com/flutter/flutter/pull/69702/files/89665643e5c7663bdd9266f45f69296e2acc2d2e#diff-4df0ac240ada8b57bf4ca7a48a0c64f3fa8efefcfb4618abd72d7b51a8fe06a6R2960

I don't think so, because the position of the TabBarView would be incorrect.

Reply:
I just checked out the latest master and copied in this test (without your other changes) and all tests (including this one) were passing. So, it's probably not testing what you think it is testing. If this is a regression test, it should be failing without your other changes.